Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -178,6 +179,21 @@ private void OnBeingPulledAttempt(EntityUid uid, CuffableComponent component, Be
                 args.Cancel();
         }
 
+        private void OnBuckleAttemptEvent(EntityUid uid, CuffableComponent component, ref BuckleAttemptEvent args)
+        {
+            // if someone else is doing it, let it pass.
+            if (args.UserEntity != uid)
+                return;
+
+            if (!TryComp<HandsComponent>(uid, out var hands) || component.CuffedHandCount != hands.Count)
+                return;
+
+            args.Cancelled = true;
+            var message = Loc.GetString("handcuff-component-cuff-interrupt-buckled-message");
```

Comment:
Either make the message vary when you are buckling/unbuckling or only display it when unbuckling (BuckleAttemptEvent raises both in TryBuckle and Unbuckle, with args.Buckling to differentiate between them)